app/cluster_data.go

```diff
@@ -17,6 +17,9 @@ func addLocalCluster(embedded bool, adminName string, management *config.Managem
 		Spec: v3.ClusterSpec{
 			Internal:    true,
 			DisplayName: "local",
+			ClusterSpecBase: v3.ClusterSpecBase{
+				DockerRootDir: "/var/lib/docker",
```

Comment:
We don't need to set it here if it is the default value

Reply:
If the crd object is created by the UI, it will have the default value, when the crd object create by the controller directly, it doesn't have the default value

Reply:
Do we really have any idea if the docker root dir of the local cluster is `/var/lib/docker`? This iseems like q too much of an assumption. What if it is legitimately _not_ that?

Reply:
`/var/lib/docker` is just the default value, this value is the same as the cluster user created in UI. Because when creating the local cluster, we don't know the docker root, so we give a default value, allow user could edit it later.

Reply:
talked to @ibuildthecloud about this. can you make this a setting with /var/lib/docker as the default? that at least will let people deploying rancher in HA mode to set the corresponding environment variable to seed the setting and make this work out of the box.
test that this flow works end-to-end

Reply:
Hi @cjellick Hi, the dockerRootDir setting is just for local cluster right? If we add this one in setting, the dockerRootDir for local cluster could be configured in two places, setting and cluster spec, they may be configured with different values, may cause more problem. 

This dockerRootDir just used in logging now, can we remove it? And the user could update rancher-logging app's answers if they want to change the default dockerRootDir.

Reply:
This would be just for the local cluster yes, so the setting could/should be called localClusterDockerRootDir.
We shouldn't remove this field at this time

Reply:
Sure, this setting is just for initial setup.